<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShiftSelect</class>
 <widget class="QDialog" name="ShiftSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelNameTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name and Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStudent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Student</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIsBlocked">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>70</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Is Blocked</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAssign">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual Assign</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUnassign">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual Unassign</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIsManual">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Is Manual</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUnsetManual">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Unset Manual</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBlock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual Block</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUnblock">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual Unblock</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBlockReason">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Block Reason</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChangeName">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChangeTime">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChangeBlockReason">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Block Reason</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetManual">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Manual</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
